### PR TITLE
PULSE-365 staging

### PR DESCRIPTION
**Pull request by brianlindsy**

### CODE REVIEW

**Review by kekey1 — APPROVED** [review]

File: src/app/common/common.service.js [hunk 1]
```diff
@@ -63,6 +64,12 @@
         function clearToken () {
             delete($localStorage.jwtToken);
         }
+
```
Comment (andlar) [inline src/app/common/common.service.js:67]: Is there a test for this function?

File: src/app/review/components/acf_patient_list/acf_patient_list.directive.js [hunk 2]
```diff
@@ -30,11 +30,12 @@
         return directive;
 
         /** @ngInject */
-        function AcfPatientListController($log, $timeout, $uibModal, commonService, QueryQueryTimeout) {
+        function AcfPatientListController($log, $filter, $timeout, $uibModal, commonService, QueryQueryTimeout) {
```
Comment (andlar) [inline src/app/review/components/acf_patient_list/acf_patient_list.directive.js:33]: I don't think $filter is needed here

File: src/app/search/components/patient_stage/patient_stage.html [hunk 3]
```diff
@@ -40,25 +40,25 @@ <h2 class="modal-title" id="titleLabel">Patient Staging</h2>
           <td rowspan="{{ location.results.length }}" href ng-click="vm.selectAll(location)">{{ location.location.name }}</td>
           <td href ng-click="location.results[0].selected = !location.results[0].selected" ng-bind-html="vm.displayNames(location.results[0].patientRecordName)"></td>
           <td href ng-click="location.results[0].selected = !location.results[0].selected">{{ location.results[0].gender.description }}</td>
-          <td href ng-click="location.results[0].selected = !location.results[0].selected">{{ location.results[0].dateOfBirth | ehealthdate }}</td>
+          <td href ng-click="location.results[0].selected = !location.results[0].selected">{{ vm.convertDobString(location.results[0].dateOfBirth) }}</td>
           <td href ng-click="location.results[0].selected = !location.results[0].selected">{{ location.results[0].ssn }}</td>
-          <td>
+          <td align="center">
```
Comment (andlar) [inline src/app/search/components/patient_stage/patient_stage.html:45]: align="center" isn't proper HTML5, but the CSS solutions are complicated and messy, so probably not worth dealing with right now

**Review by andlar — APPROVED** [review]
There are some comments here, but nothing I'd consider a blocker. @brianlindsy if you want to address my comments feel free, but if not that's fine too; just merge in the PR and we'll press.

**Pull request merged**